indico/modules/events/registration/models/registrations.py

```diff
@@ -249,6 +249,19 @@ class Registration(db.Model):
         default=False
     )
 
+    #: Whether the person is allowed to modify their registration
+    exceptional_modification_allowed = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
```

Comment:
This isn't needed, you can use the timestamp (default `None` == not allowed)

Reply:
Thanks, I removed this field and updated the logic to work with only the timestamp.